### PR TITLE
wix-ui-codemod package

### PR DESCRIPTION
**Pull request by simast**
Related to: https://github.com/wix/wix-style-react/issues/4207

Added a new `wix-ui-codemod` package with one codemod for migrating WSR icon imports from `wix-style-react/new-icons` to `wix-ui-icons-common`.

Users don't need to install this and can run using `npx wix-ui-codemod <transform> <path>` (similar to how other popular codemod packages work).

See README file for details: https://github.com/wix/wix-ui/blob/91b89528464834d7e51197354caea201045a36bb/packages/wix-ui-codemod/README.md

@shlomitc FYI

### CODE REVIEW

File: packages/wix-ui-codemod/.gitignore [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+target
+maven
+package-lock.json
+coverage
+typings
+.history
+.idea
+*.iml
+test/e2e/screenshots
+.yo-rc.json
```
Comment (argshook) [inline packages/wix-ui-codemod/.gitignore]: could also add `yarn.lock`, cause who know who'll be touching this in future

File: packages/wix-ui-codemod/package.json [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+  "jest": {
+    "transform": {
+      "^.+\\.tsx?$": "ts-jest"
+    },
+    "testPathIgnorePatterns": [
+      "<rootDir>/dist/",
+      "<rootDir>/node_modules/"
+    ]
+  },
+  "yoshi": {}
```
Comment (argshook) [inline packages/wix-ui-codemod/package.json]:
```suggestion
```

File: packages/wix-ui-codemod/src/wix-style-react/__testfixtures__/icons-common.input.js [hunk 3]
```diff
@@ -0,0 +1,9 @@
+import Add from 'wix-style-react/new-icons/Add';
+import { Check, More } from 'wix-style-react/new-icons';
+import Move from 'wix-style-react/new-icons/system/Move';
+import * as Icons from 'wix-style-react/new-icons';
```
Comment (argshook) [inline packages/wix-ui-codemod/src/wix-style-react/__testfixtures__/icons-common.input.js]: try also with renaming
`import { Icon as BestIcon } from 'wix-style-react/new-icons/Icon';`

Comment (argshook) [reply packages/wix-ui-codemod/src/wix-style-react/__testfixtures__/icons-common.input.js]: on a side note, there will likely be projects with ES5, maybe worth adding codemod/test for that too?

Comment (simast) [reply packages/wix-ui-codemod/src/wix-style-react/__testfixtures__/icons-common.input.js]: done, also added CommonJS `require('...')` transformation.

**Pull request merged**